### PR TITLE
Update URLs in feed SDK

### PR DESCRIPTION
**Pull request by cpeel**
This is just to update the URLs in this document. [Task 1927](https://www.pgdp.net/c/tasks.php?action=show&task_id=1927)

You can check it out in the [update-feed-sdk](https://www.pgdp.org/~cpeel/c.branch/update-feed-sdk/faq/feeds_sdk/) sandbox.

### CODE REVIEW

File: faq/feeds_sdk/index.php [hunk 1]
```diff
@@ -49,14 +49,12 @@
 files for Project Gutenberg's Distributed Proofreaders are created as 
 well as focusing mostly on how to implement these XML feeds into your 
 site.</p>
-<p>If you are further interested in XML you may want to look into the 
+<p>If you are further interested in XML and RSS you may want to look into the 
 follow sites as well:</p>
```
Comment (srjfoo) [inline faq/feeds_sdk/index.php:53]: "following" sites (I realize that was in the original)

**Comment by chrismiceli** [comment]
Looks good to me.im guessing localization isn't on this page yet. Out of curiosity, do we have an idea how many people use the feeds?

**Review by srjfoo — APPROVED** [review]

File: faq/feeds_sdk/index.php (hunk 1 above)
Comment (cpeel) [reply faq/feeds_sdk/index.php]: Good catch - fixed.

**Comment by cpeel** [comment]
> Looks good to me.im guessing localization isn't on this page yet. Out of curiosity, do we have an idea how many people use the feeds?

I'm dubious of the value of this page, but thought we might as well fix the URLs after Sharon found them. According to our web logs, `feeds/backend.php` was hit 8,541 times in August -- so someone still uses it.

**Pull request merged**